### PR TITLE
Two lines and next event in the channel selector list

### PR DESCRIPTION
**Pull request by Dima73**

### CODE REVIEW

**Comment by Dima73** [comment]
1_0_16_51D1_C96_3_EB0000_0_0_0_20200929221322

File: data/setup.xml [hunk 1]
```diff
@@ -54,7 +54,7 @@
 		<item level="0" text="Alternative numbering mode" description="When enabled, channel numbering will start at '1' for each bouquet.">config.usage.alternative_number_mode</item>
 		<item level="2" text="Hide number markers" description="When enabled, number markers will be hidden.">config.usage.hide_number_markers</item>
 		<item level="2" text="Show channel numbers in channel selection" description="When enabled, show channel numbers in the channel selection screen.">config.usage.show_channel_numbers_in_servicelist</item>
-		<item level="2" text="Show two lines per entry" description="Show service name and event description among each other instead of side by side.">config.usage.servicelist_twolines</item>
+		<item level="2" text="Show two lines per entry" description="Show service name and now or next event description among each other instead of side by side.">config.usage.servicelist_twolines</item>
```
Comment (WanWizard) [inline data/setup.xml]: Better is:
````
In the channel selection screen,  show the next event description below the channel name and the current event description."
````

**Comment by Dima73** [comment]
There are three options here:
1) line -->service name / current event
2)  line -->service name
     line -->current event
3) line --> service name / current event
    line -->next event

Maybe use more detailed text?

**Comment by WanWizard** [comment]
Ah, ok. Maybe something like:
````
Show the service information on two lines in the channel selection screen. You can choose between the current event description below the channel name, or the current event description next to the channel name, and the next event description on the second line.
````
?

**Pull request merged**